db/migrate/20240111105304_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] **[OPTIONAL]** This should be of type `t.text` so that it can save more text, as `t.string` has lower storage capability.